gcc/testsuite/rust/compile/macro_call_statement.rs

```diff
@@ -0,0 +1,11 @@
+fn f() {}
+
+macro_rules! panic {
+    () => (
+        crate::f()
+    );
+}
+
+fn main() {
+    panic!();
+}
```

Comment:
Is this test being part of this PR intentional?

Reply:
yes, it's fixed by the code change I just posted a comment pointing to.